aeron-driver/src/main/java/io/aeron/driver/Configuration.java

```diff
@@ -1693,7 +1689,8 @@ public static void validateInitialWindowLength(final int initialWindowLength, fi
     {
         if (mtuLength > initialWindowLength)
         {
-            throw new ConfigurationException("initial window length must be >= to MTU length: " + mtuLength);
+            throw new ConfigurationException(
+                "initial window length (" + initialWindowLength + ") must be >= MTU length (" + mtuLength + ")");
```

Comment:
`"mtuLength=" + mtuLength + " > initialWindowLength=" + initialWindowLength ` is easier to reason about given check.

Reply:
I've changed to this, but I've modified the C code to have the same error format as it simplifies the error checking in the system tests and results in a more consistent experience for users.  However in the C driver the naming convention won't match.